lib/ansible/module_utils/basic.py

```diff
@@ -2172,6 +2172,14 @@ def get_buffer_size(fd):
 
         return buffer_size
 
+    def has_setfacl(self):
```

Comment:
I think we should already be able to use typing in this module.
```suggestion
    def has_setfacl(self) -> bool:
```

Reply:
This function is not used anywhere.